script/conditional-install

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+require 'digest'
+
+def verbose command
+  puts("--> #{command}") || system(command) || fail("Failed: #{command}")
+end
+
+# Why not read node_modules/.package-lock.json? Because it's not the same.
```

Comment:
I confuse here. What is .package-lock.json?

Reply:
It's a file that `npm` itself creates, which _you would think_ corresponds to the `package-lock.json` that was used to populate `node_modules`… but it's not. I left that comment for the next person who stumbles on this script and thinks "why not just use that?".

Reply:
I checked my local node_modules folder but did not see any .package-lock.json file. Is it something in NPM 7 (I am using 6).

Reply:
I don't think so; I'm using 6.14 on this machine and it's there.